source/grid/grid_tools.cc

```diff
@@ -859,13 +859,13 @@ namespace GridTools
 
 
   template <int dim, int spacedim>
-  void
-  invert_all_negative_measure_cells(
+  std::size_t
+  invert_cells_with_negative_measure(
     const std::vector<Point<spacedim>> &all_vertices,
     std::vector<CellData<dim>> &        cells)
   {
     if (dim == 1)
-      return;
+      return 0;
     if (dim == 2 && spacedim == 3)
       Assert(false, ExcNotImplemented());
```

Comment:
@bangerth since the determinant is always positive for codim 1 or codim 2 we don't have a good way to untangle cells in that case. We should just document that this function always throws an exception outside of codim 0.

Reply:
I added a comment with what you suggested, I will push the commits in just a few minutes!